src/test/Homepage.test.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import Homepage from '../components/Homepage/Homepage';
+import { fetchStockData } from '../redux/Stocks/StocksSlice';
+import '../components/Homepage/Homepage.css';
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(),
+  useDispatch: jest.fn(),
+}));
+
+jest.mock('./Homepage.css');
+
+jest.mock('react-router-dom', () => ({
+  useNavigate: jest.fn(),
+}));
```

Comment:
[ Optional Changes ]
The previous reviewer did not point these out. But I highly recommend you work on them 👍🏽

- [ ] The test is failing on this file. See the screenshot below and see what you could do about it.

<img width="550" alt="Screenshot 2023-06-16 at 5 52 17 PM" src="https://github.com/emarkees/stocks/assets/50941074/c11ec966-a2b4-4d64-b080-b13a1f8bc6ac">

Reply:
@AmaduKamara can we have a zoom call...